apps/frontend/src/components/global/upload_tabs/tenable/FileList.vue

```diff
@@ -0,0 +1,262 @@
+      result.totalChecks = this.formatNumberOfScans(result.totalChecks);
+      result.startTime = this.epochToDate(result.startTime);
+      result.finishTime = this.epochToDate(result.finishTime);
+      this.executions.push(result);
+    });
+    this.loading = false;
+    SnackbarModule.notify(
+      'Successfully queried Tenable.sc for available scan results'
+    );
+  }
+
+  async loadResults() {
+    this.loading = true;
+
+    const files = this.selectedExecutions.map(
+      async (execution: Partial<ScanResults>) => {
+        if (execution.id) {
+          if (execution.status === 'Running') {
+            SnackbarModule.failure(
+              `Scan ${execution.id} hasn't finished, wait until completed before loading for viewing.`
+            );
+          } else {
+            const resultData = await new TenableUtil(
+              this.tenableConfig
+            ).getVulnerabilities(execution.id);
+            if (resultData) {
+              // Check is this scan is already loaded
+              let isLoaded = false;
+              const loadedFiles = InspecDataModule.allEvaluationFiles;
+
+              // We need to check if loaded zip file contained multiple scans, where they are
+              // loaded as [scanId]-[ReportHost] (i.e. 9214-mitre-saf-rhel8-mitre.org)
+              // if they are, just use the scanId for the Map key. If the scan contains a single
+              // scan it is loaded as [scanId].nessus (i.e. 9213.nessus)
+              const loadedMap = new Map(
+                loadedFiles.map((obj) => [
+                  obj.filename.indexOf('-') != -1
+                    ? obj.filename.substring(0, obj.filename.indexOf('-'))
+                    : obj.filename.substring(0, obj.filename.indexOf('.')),
+                  obj.uniqueId
+                ])
+              );
+
+              isLoaded = loadedMap.has(execution.id);
+              if (!isLoaded) {
+                try {
+                  const textFile: FileLoadOptions = {
+                    filename: execution.id + '.nessus',
+                    data: resultData
+                  };
+                  // .loadFile evaluates to data if file is not provided
+                  return await InspecIntakeModule.loadFile(textFile);
+                } catch (err) {
+                  SnackbarModule.failure(String(err));
+                }
+              }
+            } else {
+              SnackbarModule.failure(
+                'Attempted to load an undefined execution'
+              );
+              throw new Error('Attempted to load an undefined execution');
+            }
+          }
+        }
+      }
+    );
+    await Promise.all(files);
+    this.loading = false;
+    this.$emit('got-files', files);
+  }
+
+  async mounted() {
+    await this.updateSearch();
+  }
+
+  epochToDate(date: string): string {
+    return Number(date) !== -1
+      ? new Date(Number(date) * 1000).toLocaleString('en-US', {
```

Comment:
is the reason we're supplying a specific locale due to needing to supply the options as well?  is it possible to provide like an empty string/undefined/null/whatever such that it falls back to default behavior (i.e. uses the host's locale)?